src/pages/assessmentModule/AssessmentModule.jsx

```diff
@@ -182,6 +199,21 @@ const AssessmentModule = ({ type }) => {
   };
 
   const handleSave = (assessment) => {
+
+    const newTaskName = assessment.task_title;
+      // Check if the task with the same task_title already exists
+      const isDuplicateTask = assessmentList.some(
+        (existingAssessment, index) => index !== 0 && existingAssessment.task_title === newTaskName
```

Comment:
Strip the white spaces and check .
Just in case
JS task and    JS    Task should be considered as duplicate only

Reply:
I have changed